app/quality-check/get-quality-checked-payment-requests.js

```diff
@@ -1,5 +1,5 @@
 const db = require('../data')
-const { getManualLedgerRequests } = require('../manual-ledger')
+const getManualLedgerRequestCheck = require('../manual-ledger/get-manual-ledger-requests')
```

Comment:
Move this function into `manual-ledger/index/` and call it from there -- may need to change tests

Reply:
There was circular referencing error during runtime, hence, function is called outside index page to avoid circular referencing.

Reply:
Hhmmmmm interesting... definitely something we need to flag, can you create tech debt ticket to look into this?

Reply:
Seen on line 13 there's another magic number for `categoryId: 2` -- add the constant module and use it